wikipedia/scripts/digobs.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+import sys
+import subprocess
+import logging
+
+def git_hash(short=False):
+    if short:
+        return subprocess.check_output(['git', 'rev-parse', '--short', 'HEAD']).decode().strip()
+    else:
+        subprocess.check_output(['git', 'rev-parse', 'HEAD']).decode().strip()
+
+def get_loglevel(arg_loglevel):
```

Comment:
So it turns out that argparse has a cool feature where you can pass in a function that converts a string to an object and get that in your args object.  So, in theory we could pass in type=get_loglevel and it should put the loglevel in the args.

Reply:
OK. I've done this.